### PR TITLE
Add docker compose support

### PR DESCRIPTION
**Pull request by wanghouqi**
test: passed the unittest

Signed-off-by: wanghouqi <wanghouqi@vip.qq.com>

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+version: "3.9"
+services:
```
Comment (syan10) [inline docker/docker-compose.yml:2]: this version is for compose version? please add a comment

File: docker/.env [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+COUCHDB_DB=ehsm_kms_db
+
+# docker image name
+IMAGE_NAME_EHSM_BASE=ehsm_base
+
+# eHSM version
+vsn_major=0
+vsn_minor=2
+vsn_patch=0
+EHSM_VERSION_STR=${vsn_major}.${vsn_minor}.${vsn_patch}
```
Comment (syan10) [inline docker/.env:17]: is there some way to get this version automatically?

File: docker/.env [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# host ip
```
Comment (syan10) [inline docker/.env:1]: update the build-instructions doc to install the dependencies for the docker-compose

File: docker/docker-compose.yml (hunk 1 above)
Comment (wanghouqi) [reply docker/docker-compose.yml:2]: no,It is docker compose file version.
https://docs.docker.com/compose/compose-file/

File: docker/.env (hunk 3 above)
Comment (wanghouqi) [reply docker/.env:1]: ok

File: docker/.env (hunk 2 above)
Comment (wanghouqi) [reply docker/.env:17]: About the version,We need to discuss it .

**Review by syan10 — APPROVED** [review]

**Review by yang8621 — APPROVED** [review]

**Review by HawieLong — APPROVED** [review]

**Pull request merged**